modules/test/playwright/tests/journal-web/pages/JournalEditArticlePage.ts

```diff
@@ -168,19 +168,25 @@ export class JournalEditArticlePage {
 	}
 
 	async fillFriendlyURL(friendlyURL: string) {
-		await this.friendlyURLInput.fill(friendlyURL);
+		await fillAndClickOutside(
+			this.page,
+			this.friendlyURLInput,
+			friendlyURL
+		);
 	}
 
-	async createBasicArticleWithFriendlyURL(site, page, articleTitle?: string) {
+	async createBasicArticleWithFriendlyURL(site, structureTitle?: string) {
 		await this.journalPage.goto(site.friendlyUrlPath);
 		await this.journalPage.goToCreateArticle(
-			articleTitle || 'Basic Web Content'
+			structureTitle || 'Basic Web Content'
```

Comment:
could we chnage this to structureName to be consistent with goToCreateArticle() method?

Reply:
renamed the variable, Thanks